CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
Not sure, didn't think they were needed. Commit history should explain changes for developers